### DISCUSSION TITLE
Only show git-related stuff when in a repo?

### DISCUSSION THREAD

**Discussion by dm17** [starship/starship#2164] [General]
I like the git status stuff in the prompt, but I'd prefer it wasn't there when I'm not in a repository directory. Looked over the doc... Is there an option for this?
Thanks!

**Comment by dm17** [comment]
Figured it out... Can delete this.

**Reply by o6uoq** [reply]
What was the fix?

**Reply by dm17** [reply]
Accidentally initialized the dir for git... Just check if you have a ~/.git in there
